src/docs/product/session-replay/performance-overhead.mdx

```diff
@@ -24,9 +24,26 @@ Sentry's Session Replay SDK takes several measures to avoid negatively impacting
 - The SDK is built to gracefully downgrade when needed. In the event several thousand DOM mutations happen at once, the SDK will disable recording to make sure recording the website doesn't further degrade user experience. The [mutation limit values are configurable](/platforms/javascript/session-replay/configuration/#mutation-limits).
 
-## Performance Overhead in Sentry
+## Factors in Performance Overhead
 
-While the performance overhead of our client SDK will be imperceptible to end-users most of the time, it can vary based on the complexity of your application. If an applications has a large DOM and numerous DOM node mutations, it will have a higher overhead than simpler, mostly static sites. The only way to get accurate metrics is to measure performance overhead yourself.
+While the performance overhead of our client SDK will be imperceptible to end-users most of the time, it can vary based on the complexity of your application. If an applications has a large DOM and numerous DOM node mutations, it will have a higher overhead than simpler, mostly static sites.
+
+There are different stages in the lifecycle of a webpage, each with different performance characteristics. Here is a list of stages in the lifecycle, and the most important performance metrics to measure.
+
+1. Pageload
+   - [SDK bundle size is something we optimize for](https://blog.sentry.io/sentry-bundle-size-how-we-reduced-replay-sdk-by-35/)
+   - The size of your initial html page will affect how long the initial snapshot that the SDK takes, and has to send over the wire. Using your browser devtools, filter the Network panel to show "type=document" and look at the size to get a sense of the impact on your site. For example, if you show a loading screen on a single-page-app then the overhead is negligable.
+2. User scrolling & consumption
+   - Mouse-move, & hover events are incredibly small, overhead is negligable.
+3. Dynamic content & DOM Mutations
+   - The frequency of DOM mutations is highly application specific. If there are no mutations happenning then you will not see any replay data transmitted from the SDK to Sentry. On the other hand if you have JavaScript timers that updates the HTML on a regular basis, you will see a steady stream of HTML changes sent on the network.
+   - The size of DOM mutations is also highly application specific. More DOM nodes added/removed/changed at a single time will increase the time it takes to parse and minify the data. The SDK has [mutation limits](https://docs.sentry.io/platforms/javascript/session-replay/configuration/#mutation-limits) to mitigate a negative user experience: changing 750 nodes will emit a warning breadcrumb into the replay, while emitting 10,000+ nodes will cause the replay to end.
```

Comment:
```suggestion
   - The size of DOM mutations is also highly application-specific. More DOM nodes added/removed/changed in an instance will increase the time it takes to parse and minify the data. The SDK has [mutation limits](https://docs.sentry.io/platforms/javascript/session-replay/configuration/#mutation-limits) to mitigate a negative user experience: changing 750 nodes will emit a warning breadcrumb into the replay, while emitting 10,000+ nodes will cause the replay to end.
```

Reply:
original was `at a single time`

It's not really per-instance though. Updated to `at a time` , how does that sound?

Reply:
I like at once!